packages/ui-menu/src/Menu/MenuItem/index.tsx

```diff
@@ -187,8 +187,6 @@ class MenuItem extends Component<MenuItemProps, MenuItemState> {
         return 'menuitemcheckbox'
       case 'radio':
         return 'menuitemradio'
-      case 'flyout':
-        return 'button'
```

Comment:
A Menu having a `button` not a `menuitem` was driving NVDA crazy. NVDA really seems to be the most strict of them all